sdks/go/pkg/beam/io/textio/textio.go

```diff
@@ -104,7 +104,7 @@ func readFn(ctx context.Context, filename string, emit func(string)) error {
 			break
 		}
 		if err != nil {
-			return (err)
```

Comment:
I'm a little surprised this change shouldn't be submitted?

Reply:
Whoops, the Do Not Merge was from an old thing I removed. Forgot to remove it from the title and post.